### DISCUSSION TITLE
Mountainduck 5

### DISCUSSION THREAD

**Discussion by rrikerCH** [iterate-ch/cyberduck#15751] [Q&A]
Hey, is there a public timeline for MD 5 or is there a way to sign up for updates? I'm looking to use it for the SMB support. And maybe you can answer this related question at the same time. 

We have files from and old DAM server that have been archived to 2 locations stored in folders by year, one is a BLOB and one is an SMB volume depending on the region. We get regular requests to retrieve files, so I'm looking at ways to speed up the abysmal smb connection and no previews from the BLOB. 

I'm first solving for the SMB volume because I realized I could connect via Cyberduck to transfer files and it is SO much faster. We really need to be able to browse the image thumbnails though, so I'm currently connecting to the volume via smb and using an old version of Adobe Bridge (12.0.4 for the cache export functionality) to create a cache on the shared volume for each year folder. I'm wondering if a MD mounted volume could be accessed via Bridge. It looks like it should work by navigating to the mount location.

Does this seem like a viable solution to our issue?

**Comment by tatze96** [comment]
Regarding the Mountain Duck 5 release:
We don't have a specific release date for the Mountain Duck 5 major update yet, but it is planned for the second quarter of this year.

Regarding the thumbnails:
Cyberduck can't display thumbnails, but drives mounted within Finder or Explorer through Mountain Duck can show thumbnails and file previews.

Regarding Adobe Bridge:
Based on my research, Adobe Bridge supports network drives, so it should be able to access drives mounted through Mountain Duck without issues.

**Reply by f8dca** [reply]
Hi, I'm also waiting for the MB v5 release for the SMB support. Any update would be appreciated.

G.

**Reply by dzg** [reply]
+1

**Reply by rogerlovato2** [reply]
Support for the SMB protocol is already available in the latest version of Mountain Duck, but it is very poor and slow.

**Reply by eyemack** [reply]
So MD5 is a year late. Any news on this?

**Comment by vlbosch** [comment]
@tatze96 Hi! I was wondering whether you have an update on the release or timeline for Mountain Duck 5? I just noticed the Cyberduck 9.0 release, so was hoping that MD5 ia also coming soon. :-)

**Reply by dkocher** [reply]
We are working on it.

**Reply by devnoname120** [reply]
@dkocher Will I get Mountain Duck 5 for free if I buy Mountain Duck 4 right now? I'm gauging whether to wait for the new release or not before buying.

**Reply by dkocher** [reply]
For future major versions, we provide upgrade discounts depending on the date of purchase.

**Comment by IanJSaul** [comment]
Wish we could get some updates on this SMB support.  I know it's an odd request, but it potentially fixes some FUSE mounting issues I'd had.

**Reply by dkocher** [reply]
Mountain Duck does not require or make use of FUSE.

**Reply by IanJSaul** [reply]
Yes, I assumed it did not. The prior solution I was using was FUSE-based.

That's what led me to consider replacing it with Mountain Duck.

Thanks, -Ian

**Comment by jackklink** [comment]
How's that V5 coming along :) Excited for SMB support.

**Reply by rogerlovato2** [reply]
Support for the SMB protocol is already available in the latest version of Mountain Duck, but it is very poor and slow.

**Reply by aryanrajseo** [reply]
Hi, i don't see SMB support in latest Mountain duck, could you share a screenshot?

**Comment by dkocher** [comment]
We have pushed a pre-release build of Mountain Duck 5 with a new "Integrated" connect mode using File Provider API on macOS and Cloud Files API on Windows [^1]. The build can be used with a validation registration key for version 4. Choose _Preferences → Update → Beta → _Check for Update…_ in the Mountain Duck status bar menu to update.

[^1]: https://blog.cyberduck.io/2025/07/14/mountain-duck-5-public-beta/

**Comment by llFawkes17ll** [comment]
Does the moutainduck 5 update bring native support for windows on arm?

**Reply by dkocher** [reply]
Discussed in #17353.
